packages/payload/src/admin/components/elements/Tooltip/index.tsx

```diff
@@ -42,18 +49,31 @@ const Tooltip: React.FC<Props> = (props) => {
     <React.Fragment>
       <aside
         aria-hidden="true"
-        className={['tooltip', className, 'tooltip--position-top'].filter(Boolean).join(' ')}
+        className={[
+          'tooltip',
+          className,
+          messageIsLong ? 'tooltip--position-stretch-top' : 'tooltip--position-top',
+        ]
+          .filter(Boolean)
+          .join(' ')}
         ref={ref}
+        title={getTitleAttribute(children)}
       >
-        {children}
+        <div className="tooltip-content">{children}</div>
       </aside>
 
       <aside
-        className={['tooltip', className, show && 'tooltip--show', `tooltip--position-${position}`]
+        className={[
+          'tooltip',
+          className,
+          show && 'tooltip--show',
+          messageIsLong ? 'tooltip--position-stretch-top' : `tooltip--position-${position}`,
```

Comment:
Let me know when you want to boot this up and we can take a look visually together 👍